과제3/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html lang="ko">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>디자인과제3</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+<link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+<link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap" rel="stylesheet">
+<link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div class="box">
+        <div class="header">
+            <img src="navigation.png" class="img-nav" alt="">
+            <p class="title">Machiatto</p>
+            <img src="cart.png" class="img-cart" alt="">
+        </div>
+            <button class="heart">
+                <img src="heart.png" width="24px" height="24px" alt="">
+                <p class="heart248">248</p>
+            </button>
+        <div class="main">
+            <img src="coffee-main.png" class="img-main" alt="">
+            <p class="Machiatto">Machiatto</p>
+            <p class="text-1">Ma Kira To (Machiatto), the espresso add two<br>
+                tablespoons of milk into a cup of macciato. Macdo<br>
+                is the brand mark meaning in Italian, so a symbol<br>
+                of sweet imprint.</p>
+            <p class="price">$ 5.88</p>
```

Comment:
p 태그는 주로 문단을 표현할 때 많이 쓰는 태그입니다. 
다음 상황에서는 p 태그보다 span 태그를 사용하는 것이 좋을 것 같습니다.
```suggestion
            <span class="price">$ 5.88</span>
```